Sources/BraintreePayPalNative/Public/BraintreePayPalNative/BTPayPalNativeClient.swift

```diff
@@ -48,6 +48,7 @@ import PayPalCheckout
                                                                        environment: order.environment)
 
                 PayPalCheckout.Checkout.set(config: payPalNativeConfig)
+                PayPalCheckout.Checkout.showsExitAlert = false
```

Comment:
I thought the option to hide the modal was supposed to be part of the merchant PayPalRequest, are we just always hiding it instead?

Reply:
I guess that would be a question for the team. I was under the impression that we always want to hide it.

Reply:
We always want to hide it!